sre-operational-fabric.md

```diff
@@ -0,0 +1,71 @@
+# (Remote) SRE - Rackspace Private Cloud
+
+As a Rackspace Private Cloud (RPC) Operational Fabric SRE, you will help
+support our organizational vision: To become the preferred brand for consuming
+complex open source infrastructure, as-a-Service, delivering industry-leading
+simplicity, efficiency, and reliability.
+
+Our team supports this mission by improving
+[observability](https://en.wikipedia.org/wiki/Observability) (i.e. insight
+into) and [controllability](https://en.wikipedia.org/wiki/Controllability)
+(i.e. state mutation) of cloud deployments located in datacenters throughout
+the world.
+
+## Role
+
+Site Reliability Engineering (SRE) is an engineering discipline that combines
+software and systems engineering to build and run large-scale, massively
+distributed, fault-tolerant systems. SRE ensures that Rackspace's managed
+service offerings & customer deployments have reliability and uptime
+appropriate to users' needs and a fast rate of improvement while monitoring and
+validating capacity and performance. Act as a Technical Lead for a portfolio of
+managed service deployments. Focused on reliability, scalability and the
+development of automation to manage a set of repetitive tasks at scale.
+
+## About Us
+
+- **Team Size**: Currently 8 senior developers/engineers and a technical manager
+- **Hours**: We have flexible working hours and aim for a median of 35-40 hours a week
+- **On-Call**: No, we have a 24x7 ops team and escalations (uncommon) go to management
+- **Location**: US/UK work from home with offices in US/UK (most states supported)
+- **Travel**: About once or twice a year for a conference or team gathering
+
+## Responsibilities
+
+- Help develop and maintain *observability*/controllability of customer environments
```

Comment:
Why is observability emphasized? I guess that's what we've been working on most lately

Reply:
Yeah